--- gtypist-2.8.3-orig/src/error.c	2009-04-07 18:42:30.167021789 +0200
+++ gtypist-2.8.3/src/error.c	2009-04-07 18:42:46.073519275 +0200
@@ -21,11 +21,7 @@
 #include "config.h"
 #include "error.h"
 #include "script.h"
-#ifdef HAVE_LIBCURSES
-#include <curses.h>
-#else
 #include <ncurses/curses.h>
-#endif
 #include <stdlib.h>
 
 #include "gettext.h"
--- gtypist-2.8.3-orig/src/cursmenu.c	2009-04-07 18:42:30.167021789 +0200
+++ gtypist-2.8.3/src/cursmenu.c	2009-04-07 18:45:17.700752414 +0200
@@ -21,11 +21,7 @@
 #include "cursmenu.h"
 #include "script.h"
 
-#ifdef HAVE_LIBCURSES
-#include <curses.h>
-#else
 #include <ncurses/curses.h>
-#endif
 
 #include "error.h"
 #include "gtypist.h"
--- gtypist-2.8.3-orig/src/gtypist.c	2009-04-07 18:42:30.167021789 +0200
+++ gtypist-2.8.3/src/gtypist.c	2009-04-07 18:45:34.140133532 +0200
@@ -27,11 +27,7 @@
 #include <ctype.h>
 #include <signal.h>
 
-#ifdef HAVE_LIBCURSES
-#include <curses.h>
-#else
 #include <ncurses/curses.h>
-#endif
 
 #include <time.h>
 #include <errno.h>
--- gtypist-2.8.3-orig/src/script.c	2009-04-07 18:42:30.167021789 +0200
+++ gtypist-2.8.3/src/script.c	2009-04-07 18:48:07.830117125 +0200
@@ -20,11 +20,7 @@
  */
 #include "config.h"
 #include "script.h"
-#ifdef HAVE_LIBCURSES
-#include <curses.h>
-#else
-#include <ncurses.h>
-#endif
+#include <ncurses/ncurses.h>
 #include "error.h"
 
 #include <stdlib.h>
