--- gtypist-2.10.1/configure.ac.orig	2025-02-08 10:40:02.264076914 +0100
+++ gtypist-2.10.1/configure.ac	2025-02-08 10:40:27.740605562 +0100
@@ -27,7 +27,7 @@ AC_INIT([GNU Typist],
         [bug-gtypist@gnu.org], [gtypist])
 AM_INIT_AUTOMAKE([1.8.2 gnu dist-xz])
 AC_CONFIG_SRCDIR([src/gtypist.c])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
 AC_USE_SYSTEM_EXTENSIONS
 AC_CANONICAL_HOST
 
